plugins/obs-ffmpeg/ffmpeg-encoded-output.c

```diff
@@ -0,0 +1,915 @@
+/******************************************************************************
+    Copyright (C) 2019 Haivision Systems Inc.
+    Copyright (C) 2014 by Hugh Bailey <obs.jim@gmail.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#include "ffmpeg-encoded-output.h"
+
+static const char *ffmpeg_encoded_output_getname(void *unused);
+static void ffmpeg_encoded_output_destroy(void *data);
+static void *ffmpeg_encoded_output_create(obs_data_t * settings,
+					  obs_output_t * output);
+static void ffmpeg_encoded_output_stop(void *data, uint64_t ts);
+static bool ffmpeg_encoded_output_start(void *data);
+static void ffmpeg_encoded_output_data(void *data,
+				       struct encoder_packet *packet);
```

Comment:
Style:  No need to align like this.  Use two tabs to indent a line continuation.

```suggestion
		struct encoder_packet *packet);
```